backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -121,6 +134,13 @@ describe('ContributionResolver', () => {
           )
         })
 
+        it('logs the error found', () => {
+          expect(logger.error).toBeCalledWith(
+            'No information for available creations with the given creationDate=',
+            new Date('non-valid').toDateString,
```

Comment:
does the Date object not throw an exception in case of a not parsable datestring?
and how can such an invalidated Date instance convert its internal invalid date to a datestring?
is it not better to simply concatinate the 'non-valid' string to the message-string?

Reply:
it is true, I agree
it generates string "Invalid Date", will use that directly